### PR TITLE
Tests/cypress tests

### PR DESCRIPTION
**Pull request by mainlyetcetera**
## Is this a fix or a feature?
- feature
## What is the change/fix?
- adds tests for the entire app using `cypress`
## Where should the reviewer start?
- `cypress/integration/App-spec.js`
## How should this be tested?
- pull down branch (not to main initially)
- run `npm run cypress`
- observe all tests passing
## Screenshots(if appropriate)
image
## This PR is related to which issues:
- none

### CODE REVIEW

**Comment by mainlyetcetera** [comment]
Note: this pr DOES NOT yet include sad-path tests for missing data. I am working on that here in a little bit!

**Comment by mainlyetcetera** [comment]
As of this update, there are now tests looking at whether data exists or not according to whether data was given!

File: cypress/integration/App-spec.js [hunk 1]
```diff
@@ -0,0 +1,338 @@
+describe('the main user flow', () => {
+  it('should visit the page with test data', () => {
+    cy
+      .fixture('../fixtures/allMovies.json')
+      .then(data => {
+        cy.intercept('GET', 'https://rancid-tomatillos.herokuapp.com/api/v2/movies', {
+          statusCode: 200, 
+          body: data
+        })
```
Comment (CJSpohn) [inline cypress/integration/App-spec.js:9]: Glad we verified our intercepts are working currently with changing a value in the dummy data.

File: cypress/integration/App-spec.js [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+
+    cy
+      .get('div p')
+      .contains('Revenue: ')
+      .should('not.exist')
+
+    cy
+      .get('div p')
+      .contains('Runtime: ')
+  });
```
Comment (CJSpohn) [inline cypress/integration/App-spec.js:337]: Really good happy/sad path testing for our conditional rendering.

**Review by CJSpohn — APPROVED** [review]
Thanks for going back and adding the tests for making sure data doesn't show when we have 0s for budget and revenue.

**Pull request merged**